Misc/NEWS.d/next/Library/2017-12-27-20-09-27.bpo-32433.vmxsVI.rst

```diff
@@ -0,0 +1,2 @@
+The hmac module now has hmac.digest(), which provides an optimized HMAC
+digest for short messages.
```

Comment:
Is the "for short messages" part true?  does anything care how long the message is?  I'd just trim those words from the sentence. :)

Reply:
"short" for some arbitrary measurement. After all the message has to fit into memory. Anything up to 64kB should be fine. :p